go-controller/pkg/ovn/endpoints.go

```diff
@@ -261,13 +187,18 @@ func (ovn *Controller) deleteEndpoints(ep *kapi.Endpoints) error {
 			klog.V(5).Infof("Reject ACL created for load balancer: %s, %s", lb, aclUUID)
 		}
 
+		if util.ServiceTypeHasNodePort(svc) {
```

Comment:
thanks for fixing this, but it has nothing to do with external IP. This kind of stuff can you please move to another commit. Really another PR would even be better as this fixes not removing node port load balancer when we delete, so better to not hold it up by this PR.

Reply:
Ok, I will put this is a separate commit and PR.